peachjam/static/stylesheets/_global.scss

```diff
@@ -141,3 +141,7 @@ footer {
     }
   }
 }
+
+[data-user-agent*='pocketlaw'] {
```

Comment:
This would be "html[data...] .pocket law-hidden { display: none;}"

(Abbreviated because on my phone)

Reply:
Works without the html part

Reply:
Just seen the other comment, fixed both. Now it works